tests/jenkins/TestbuildManifestVar.groovy

```diff
@@ -8,13 +8,25 @@
  */
 package jenkins.tests
 
-import org.junit.Test
+import org.junit.*
```

Comment:
Should have avoided wildcard here. junit has way too many subclasses. Please fix this in next iteration

Reply:
Sure thanks.